yassersharabati/main.py

```diff
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, HTTPException
+from uuid import uuid4
+from typing import List
+from task import *
+
+app = FastAPI()
+
+
+tasks = []
+
+@app.post("/tasks/", response_model=Task)
+def create_task(task: Task):
+    task.task_id = str(uuid4()) 
+    tasks.append(task)
+    return task
+
+
+@app.put("/tasks/{task_id}/", response_model=Task)
+def update_task(task_id: str, updated_task: Task):
+    for task in tasks:
+        if task.task_id == task_id:
+            task.title = updated_task.title
```

Comment:
Is there a better way to update the task? 
What if there are 20 properties for the task? Is that good? 
I think you can iterate over the keys and update the values

Reply:
Another note.
The user must have the ability to edit one property if he wants, but in your way the user must edit all task data in the update operation.
It must be optionally available

Reply:
I have solved this point